app/javascript/elm/src/MobileSessionsFilters.elm

```diff
@@ -27,6 +30,7 @@ defaultModel =
     , isCrowdMapOn = False
     , tags = Labels.empty
     , profiles = Labels.empty
+    , timeRange = TimeRange.defaultTimeRange
     }
```

Comment:
Is the default model useful here? Cause it's supposed to be overwritten by flags anyway.

Reply:
That's a good question. Maybe we don't need a default model